com.unity.render-pipelines.universal/Runtime/2D/Light2D.cs

```diff
@@ -318,5 +330,26 @@ private void LateUpdate()
             UpdateMesh(true);
             UpdateBoundingSphere();
         }
+
+#if UNITY_EDITOR
```

Comment:
This is causing a compile error on build, because the implementation is missing in non editor compilations, but the interface is specified on the class regardless.

What's the reason for the implementation only being in editor?